### ISSUE TITLE
Project dead?

### ISSUE THREAD

**Issue by mackelito** [AngularDoc/angulardoc.github.io#47]
Just wanted to know if the project is still alive or not?

**Comment by agiledigits** [comment]
Yes it's alive and well. We are working on a cloud native version of it. Stay tuned.

**Comment by mackelito** [comment]
Ok.. just wanted to see if I should post potential bugs or not..  :)

**Comment by mackelito** [comment]
BTW... Does that mean that you will abandon the desktop app/vs code extension?

**Comment by HankBu** [comment]
hello, how is the project, it doesn't work until 2023
